svg/elements/textpath.json

```diff
@@ -0,0 +1,390 @@
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": "3",
+              "notes": [
+                "Until Safari 6, textPath was not re-render when the referenced path was changed dynamically (see <a href='https://bugs.webkit.org/show_bug.cgi?id=15799'>bug 15799</a>'"
+              ]
+            },
+            "safari_ios": {
+              "version_added": "3",
+              "notes": [
+                "Until Safari 6, textPath was not re-render when the referenced path was changed dynamically (see <a href='https://bugs.webkit.org/show_bug.cgi?id=15799'>bug 15799</a>'"
+              ]
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "href": {
+          "__compat": {
+            "support": {
+              "webview_android": {
+                "version_added": true
+              },
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": true
+              },
+              "firefox_android": {
+                "version_added": true
+              },
+              "ie": {
+                "version_added": true
+              },
+              "opera": {
+                "version_added": true
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": true
+              },
+              "safari_ios": {
+                "version_added": true
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "method": {
```

Comment:
The current versions of Firefox, Chrome, Opera, IE 11, and Edge [don't seem to support this attribute](https://codepen.io/anon/pen/xWENLL).